lib/tasks/argo.rake

```diff
@@ -261,9 +261,12 @@ namespace :argo do
   end
 
   def load_order_files(fedora_files)
+    data_path = File.expand_path('../../../fedora_conf/data/', __FILE__)
```

Comment:
Yes, understood.  I didn't have the energy to be perfect after a less than perfect day at the keyboard ;-)  This file was touched because I wanted a blank line in the `load_order` file to separate out the fixtures included for working on #289, which have been subsequently removed for several reasons, primarily because they are not representative of the argo-stage data and secondarily because it may not be good to add a new fixture data file for every bug and lastly because the spec code may suffice to test the method modifications in this PR.  But thanks for encouraging perfection and I'm open to suggestions on how to work around this if it's really important.